jenkins/Jenkinsfile

```diff
@@ -12,6 +12,24 @@ def getClusterManagerPreRelease(String branchName='master'){
   """
 }
 
+def buildClusterManagerRPM(String workspace, String version, String prerelease, String branchName='master'){
+  container('rpmbuild'){
+    sh """
+      cd && mkdir rpmbuild && cd rpmbuild
+      git clone --single-branch --branch ${branchName} https://github.com/cloudify-cosmo/cloudify-cluster-manager.git SOURCES && cd SOURCES
```

Comment:
@YoniItzhak you can use method which is part of our shared library 
```
repoCheckout("${env.GITHUB_PROJECT_URL}", "${env.PROJECT}", "${env.BRANCH}")
```

Check

Reply:
@YoniItzhak why you clone this? you already have 'cloudify-cluster-manager' dir before jobs run?
if that so just copy it

Reply:
@warolv, meaning I have it on `rpmbuild` from this stage? https://github.com/cloudify-cosmo/cloudify-cluster-manager/blob/87aeadd7ec3d41dfa29e9864e9e285a6d8e231d4/jenkins/Jenkinsfile#L135

Reply:
@mabuaisha, thanks. I think `repoCheckout` doesn't have the option of cloning a branch to a different folder, as I do here to `SOURCES`, but maybe I'm missing something.

Reply:
@YoniItzhak I think @warolv is right, since you already this project is checkout here https://github.com/cloudify-cosmo/cloudify-cluster-manager/pull/27/files#diff-6c263717a24af2836fc1795f59bdea6b8616d61bd499edb331d6c9d253478e34R45

Then you do not need to clone it and all you need to do is just reference that dir from Workspace, check how we are doing that on build our rpms https://github.com/cloudify-cosmo/cloudify-build-system/blob/master/pipelines-k8s/build-rpms/Jenkinsfile#L74 and check the method itself which is just reference the dir that contains the code that need to be built

Reply:
Thanks @mabuaisha, I changed the function to just use the already cloned repo as done here https://github.com/cloudify-cosmo/cloudify-build-system/blob/master/pipelines-k8s/build-rpms/Jenkinsfile#L11